requirements/test-integration.txt

```diff
@@ -1,7 +1,8 @@
+pytest-rerunfailures>=6.0
 simplejson
--r extras/redis.txt
--r extras/dynamodb.txt
--r extras/azureblockblob.txt
 -r extras/auth.txt
+-r extras/azureblockblob.txt
+-r extras/database-integration.txt
```

Comment:
I think this like is the only change needed here.

Reply:
The azure block blob integration tests are failing without this line, but you're right it has no place in this PR.